generator/api/template/src/test/test.js

```diff
@@ -3,40 +3,16 @@ import 'babel-polyfill';
 import assert from 'assert';
 import GitHubUser from '../lib/GitHubUser';
 
-// WARNING: don't use arrow functions with mocha describe(), it(),
```

Comment:
The reason the comment is there is to rationalize the use of `function` when the rest of the code around is using arrow functions. We can certainly make it more terse, but unless you feel it absolutely must not be there, I would prefer to keep the comment close to the code it rationalizes...